configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
Thanks, that makes it clearer. So in our case we would add an additional step in the build pipeline to tag 'latest' after building the image with the target version.